src/main/java/checker/MiLoGChecker.java

```diff
@@ -85,9 +85,18 @@ public CheckerReturn check() throws CheckerException {
      * Checks whether maximum working time was exceeded.
      */
     protected void checkTotalTimeExceedance() {
+        //Legal maximum working time per month
         TimeSpan maxWorkingTime = fullDoc.getProfession().getMaxWorkingTime();
         
-        if (fullDoc.getTotalWorkTime().compareTo(maxWorkingTime) > 0) {
+        //Sum of all daily working times (without pauses!)
+        TimeSpan totalWorkingTime = fullDoc.getTotalWorkTime();
+        
+        //Vacation and transfer corrected time
+        TimeSpan correctedMaxWorkingTime = maxWorkingTime.subtract(fullDoc.getVacation())
+                .subtract(fullDoc.getPredTransfer())
```

Comment:
Does TimeSpan subtraction work even when the result is negative? I'm unsure what the result of the % operator is in that case. (see /src/main/java/data/TimeSpan.java)

### Scenario:
Given my max working time is 40 hours and I transfer 50 from the month before (maybe transfer 50 to next month as well), then the result of this subtraction would be negative.

Reply:
Good point. The problem is that this is caused by the TimeSheet constructor itself. I will quickly fix that and commit it. Hold on.

Reply:
Should be fixed now.